### PR TITLE
test(react): write OpenGraph test code

### PR DESCRIPTION
**Pull request by ChanhyukPark-Tech**
## Overview

OpenGraph, which sets the meta tag, which is such a good component, does not have a test code, so I wrote it for safety .
when merge #77 it should fix something just import part

## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed. yes I made them

### CODE REVIEW

File: packages/react/react/src/components/OpenGraph/index.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    expect(getMeta('og:description')).toEqual('금융의 모든 것, 토스에서 쉽고 간편하게');
+  });
+  it(`should render inserted imageUrl`, () => {
+    render(<OpenGraph imageUrl="https://static.toss.im/homepage-static/newtoss/newtoss-og.jpg" />);
+    expect(getMeta('og:image')).toEqual('https://static.toss.im/homepage-static/newtoss/newtoss-og.jpg');
+  });
+  it(`should throw error when no meta tag is found `, () => {
+    render(<OpenGraph />);
+    expect(() => getMeta('og:title')).toThrowError();
+  });
```
Comment (ChanhyukPark-Tech) [inline packages/react/react/src/components/OpenGraph/index.spec.tsx:29]: I implemented it like this to test the error part, is it the right method? If you have a better way, please recommend it!

File: packages/react/react/src/components/OpenGraph/index.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    for (let i = 0; i < metas.length; i += 1) {
+      if (metas[i].getAttribute('property') === metaName) {
+        return metas[i].getAttribute('content');
+      }
+    }
+    throw new Error('cannot find your meta tag');
+  };
+  it(`should render inserted title`, () => {
+    render(<OpenGraph title="토스" />);
+    expect(getMeta('og:title')).toEqual('토스');
```
Comment (manudeli) [inline packages/react/react/src/components/OpenGraph/index.spec.tsx:16]:
```suggestion
    const title = '토스'

    render(<OpenGraph title={title} />);
    expect(getMeta('og:title')).toEqual(title);
```

File: packages/react/react/src/components/OpenGraph/index.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  });
+  it(`should render inserted description`, () => {
+    render(<OpenGraph description="금융의 모든 것, 토스에서 쉽고 간편하게" />);
+    expect(getMeta('og:description')).toEqual('금융의 모든 것, 토스에서 쉽고 간편하게');
```
Comment (manudeli) [inline packages/react/react/src/components/OpenGraph/index.spec.tsx:20]:
```suggestion
    const description = '금융의 모든 것, 토스에서 쉽고 간편하게'

    render(<OpenGraph description={description} />);
    expect(getMeta('og:description')).toEqual(description);
```

File: packages/react/react/src/components/OpenGraph/index.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  });
+  it(`should render inserted imageUrl`, () => {
+    render(<OpenGraph imageUrl="https://static.toss.im/homepage-static/newtoss/newtoss-og.jpg" />);
+    expect(getMeta('og:image')).toEqual('https://static.toss.im/homepage-static/newtoss/newtoss-og.jpg');
```
Comment (manudeli) [inline packages/react/react/src/components/OpenGraph/index.spec.tsx:24]:
```suggestion
    const imageUrl = 'https://static.toss.im/homepage-static/newtoss/newtoss-og.jpg'
    
    render(<OpenGraph imageUrl={imageUrl} />);
    expect(getMeta('og:image')).toEqual(imageUrl);
```

**Review by manudeli — COMMENTED** [review]
It's really trivial, but How about below suggestion comments to avoid chance that 2 long strings can be slightly different by human mistake not on purpose?

your intention is that OpenGraph's title is ```equal``` with Meta's title
So I want anyone cannot make mistake if new test code is required

File: packages/react/react/src/components/OpenGraph/index.spec.tsx (hunk 2 above)
Comment (ChanhyukPark-Tech) [reply packages/react/react/src/components/OpenGraph/index.spec.tsx:16]: Your suggestion is so clear
it cannot make human mistake
I will apply your suggestion 
Thank you!

File: packages/react/react/src/components/OpenGraph/index.spec.tsx (hunk 4 above)
Comment (ChanhyukPark-Tech) [reply packages/react/react/src/components/OpenGraph/index.spec.tsx]: thx!
I applied it!

File: packages/react/react/src/components/OpenGraph/index.spec.tsx (hunk 1 above)
Comment (raon0211) [reply packages/react/react/src/components/OpenGraph/index.spec.tsx:32]: I would probably return `null` when errors can happen quite often, but this is really a minor opinion.

**Review by raon0211 — APPROVED** [review]
Thanks, @ChanhyukPark-Tech !

**Pull request merged**